apps/passport-server/src/services/telegramService.ts

```diff
@@ -118,7 +132,7 @@ export class TelegramService {
     this.bot.command("start", async (ctx) => {
```

Comment:
note that because of #621, we'll have to change instances of ZKEdDSATicketPCD to ZKEdDSAEventTicketPCD